merkle/accumulator_test.go

```diff
@@ -155,7 +155,9 @@ func TestApplyBlock(t *testing.T) {
 		leaves = append(leaves, FileContractLeaf(fces[i], false))
 	}
 	var acc ElementAccumulator
-	acc.ApplyBlock(nil, leaves)
+	for i := 0; i < 5; i++ {
+		acc.ApplyBlock(nil, leaves)
+	}
```

Comment:
oh dear. I don't even *know* what happens if you add the same leaves multiple times. I think `ApplyBlock` assumes that the leaves have no index/proof, and fills them in -- so this is probably equivalent to adding 4*len(leaves) arbitrary leaves to the accumulator, then calling `ApplyBlock` once on the actual leaves.

To get the coverage we want, I think we can actually just set `acc.NumLeaves` directly; a value like 6 or 9 should work well.